torch/_dynamo/resume_execution.py

```diff
@@ -133,7 +208,7 @@ def update(instructions: List[Instruction], code_options: Dict[str, Any]):
             freevars = tuple(code_options["co_cellvars"] or []) + tuple(
                 code_options["co_freevars"] or []
             )
-            code_options["co_name"] = f"<graph break in {code_options['co_name']}>"
+            code_options["co_name"] = f"<resume in {code_options['co_name']}>"
```

Comment:
calling the resume function graph break is misleading. The code stub that calls resume is what deserves to be called graph break.